howto/download_stablelm.md

```diff
@@ -1,4 +1,4 @@
-## Downloading pretrained weights
+## Download [StableLM](https://github.com/Stability-AI/StableLM) weights
```

Comment:
Since this file is specific to StableLM now, we could add an intro note on what it is as we have for redpajama and pythia